packages/account/src/providers/provider.test.ts

```diff
@@ -2281,4 +2281,72 @@ Supported fuel-core version: ${mock.supportedVersion}.`
 
     expect(fetchChainAndNodeInfo).toHaveBeenCalledTimes(2);
   });
+
+  it('submits transaction and awaits status [success]', async () => {
+    using launched = await setupTestProviderAndWallets({
+      nodeOptions: {
+        args: ['--poa-instant', 'false', '--poa-interval-period', '1s'],
+      },
+    });
```

Comment:
```suggestion
    using launched = await setupTestProviderAndWallets();
```

These args aren't necessary if you won't be testing for `SqueezedOutStatus`.